src/main/java/com/teamc/fms/controller/FacilityController.java

```diff
@@ -0,0 +1,44 @@
+package com.teamc.fms.controller;
+
+import com.teamc.fms.entity.Facility;
+import com.teamc.fms.service.FacilityService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/v1/facilities")
+public class FacilityController {
+
+    @Autowired
+    private FacilityService facilityService;
+
+    @GetMapping("/")
+    public List<Facility> getFacilities(){
+        return facilityService.getFacilities();
+    }
+
+    @PostMapping("/")
+    public String addFacility(@RequestBody Facility facility){
+
+        facilityService.addFacility(facility);
+        return "successfully added facility";
+    }
+    @DeleteMapping("/")
+    public String deleteFacility(@RequestParam String id){
+
+        facilityService.deleteFacility(id);
+        return "successfully deleted";
+    }
+
+    @PutMapping("/")
+    public String updateFacility(@RequestBody Facility facility){
+
+         facilityService.updateFacility(facility);
+
+        return "successfully updated";
+
+    }
+
```

Comment:
-DTOs are preferred to encapsulate the data that is to be transferred between layers or processes, and use mappers to transfer between entities and DTOs.

Reply:
Yes, I agree with, like the attached image
![image](https://github.com/web-development-course-way/fms/assets/31379298/fa00bb36-71e4-40a4-b852-6a572f5371e7)